src/azure-cli/azure/cli/command_modules/appservice/tests/latest/sample_multiple_csx_function_httptrigger/HttpTrigger/run.csx

```diff
@@ -0,0 +1,23 @@
+﻿#r "Newtonsoft.Json"
+
+using System.Net;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using Newtonsoft.Json;
+
+public static async Task<IActionResult> Run(HttpRequest req, ILogger log)
+{
+    log.LogInformation("C# HTTP trigger function processed a request.");
+
+    string name = req.Query["name"];
+
+    string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+    dynamic data = JsonConvert.DeserializeObject(requestBody);
+    name = name ?? data?.name;
+
+    string responseMessage = string.IsNullOrEmpty(name)
+        ? "This HTTP triggered function executed successfully. Pass a name in the query string or in the request body for a personalized response."
+                : $"Hello, {name}. This HTTP triggered function executed successfully.";
+
+            return new OkObjectResult(responseMessage);
+}
```

Comment:
Because there are special file types *(.csx)* in this PR, our packaging test task of release pipeline cannot identify them. Please put the directory `sample_multiple_csx_function_httptrigger` under the `data` directory, such as `/latest/data/sample_multiple_csx_function_httptrigger`